From c1a90e70ea0ccd5788660e29fa5515635183f108 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Sat, 17 Feb 2018 19:16:57 +0900
Subject: [PATCH 130/909] clk: renesas: r8a77990: Add FDP1 clock

commit ec03e7d30c31d97e1a25d20be872343169267d5b from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds FDP1-0 clock to the R8A77990 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a77990-cpg-mssr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/renesas/r8a77990-cpg-mssr.c b/drivers/clk/renesas/r8a77990-cpg-mssr.c
index 5b331d82ae6d..2bafe7674ce7 100644
--- a/drivers/clk/renesas/r8a77990-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a77990-cpg-mssr.c
@@ -125,6 +125,7 @@ static const struct cpg_core_clk r8a77990_core_clks[] __initconst = {
 
 static const struct mssr_mod_clk r8a77990_mod_clks[] __initconst = {
 	DEF_MOD("3dge",			 112,	R8A77990_CLK_ZG),
+	DEF_MOD("fdp1-0",		 119,	R8A77990_CLK_S0D1),
 	DEF_MOD("scif5",		 202,	R8A77990_CLK_S3D4C),
 	DEF_MOD("scif4",		 203,	R8A77990_CLK_S3D4C),
 	DEF_MOD("scif3",		 204,	R8A77990_CLK_S3D4C),
-- 
2.17.1

